### PR TITLE
Add an option to disallow the validator creation

### LINKED ISSUE #61: Option to disable the validator creation
See https://github.com/medibloc/cosmos-sdk/issues/4

### PR DESCRIPTION
**Pull request by youngjoon-lee**
Closes #61 
This PR is based on the PR #58 (for the new cosmos-sdk).

NOTE:
This is implemented in the `x/staking/handler.go`, because the baseapp flag isn't accessible in the `ValidateBasic()` (in the `x/staking/types/msg.go`).
It means that the NotAllowed error occurs from `DeliverTx()`.

Tests:
```
$ panaceacli tx staking create-validator ..... -b block

...
logs:
- msgindex: 0
  success: false
  log: '{"codespace":"staking","code":4,"message":"not allowed"}'
...
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,4 +26,4 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
 
-replace github.com/cosmos/cosmos-sdk => github.com/medibloc/cosmos-sdk v0.37.14-internal
+replace github.com/cosmos/cosmos-sdk => github.com/medibloc/cosmos-sdk v0.37.14-internal.0.20201013062843-a5f121b3001e
```
Comment (youngjoon-lee) [inline go.mod:29]: TODO: Use a proper tag after merging https://github.com/medibloc/cosmos-sdk/pull/7

Comment (youngjoon-lee) [inline go.mod:29]: TODO: Use a proper tag after merging https://github.com/medibloc/cosmos-sdk/pull/7

**Review by cl9200 — APPROVED** [review]
:+1:

File: go.mod (hunk 1 above)
Comment (youngjoon-lee) [reply go.mod]: Done 3ab6d3d

**Pull request merged**